colab/test_nessie_iceberg_demo_nba.py

```diff
@@ -0,0 +1,123 @@
+import pytest
```

Comment:
Can you move the test-class into a separate directory?

Reply:
was thinking to move it to the same dir where `test_demo.py` is located, wdyt?

Reply:
The stuff in `nessiedemo/` is rather a self-contained package/library. Having the tests for the notebooks in the tests for the library would introduce a somewhat cyclic dependency.

Reply:
agreed. moved into its own directory